taskfile/taskfile.go

```diff
@@ -89,7 +88,6 @@ func RemoteExists(ctx context.Context, l *logger.Logger, u *url.URL, timeout tim
 
 		// If the request was successful, return the URL
 		if resp.StatusCode == http.StatusOK {
-			l.VerboseOutf(logger.Magenta, "task: [%s] Not found - Using alternative (%s)\n", alt.String(), taskfile)
```

Comment:
I’m OK with removing it because, even though we could use it for troubleshooting, I don’t think it’s used at all. I just wanted to make sure it wasn’t an oversight.